db/seeds.rb

```diff
@@ -5,3 +5,36 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+hs2500 = Machine.find_or_create_by(name: 'HS2500')
+hs2500_kits = %w(58c 66c 160c 168c 202c 210c 218c 266c 316c 508c 516c cBotV1)
+hs2500_kits.each do |kit_name|
+  kit = Kit.find_or_create_by(name: kit_name)
+  MachineKitCompatibility.find_or_create_by(machine: hs2500, kit: kit)
+end
+
+hsx = Machine.find_or_create_by(name: 'HSX')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.